cppcheck_junit.py

```diff
@@ -128,14 +140,29 @@ def generate_test_suite(errors: Dict[str, List[CppcheckError]]) -> ElementTree.E
             time=str(1),
         )
         for error in errors:
-            ElementTree.SubElement(
+            error_element = ElementTree.SubElement(
                 test_case,
                 "error",
-                type="",
+                type=error.severity,
                 file=os.path.relpath(error.file) if error.file else "",
-                line=str(error.line),
-                message=f"{error.line}: ({error.severity}) {error.message}",
+                message=f"{error.message}",
```

Comment:
Before, only `line: (severity) message` was added to the `message` attribute. Now only `message` is added.